### DISCUSSION TITLE
Welcome to Page-Replica.com Support!

### DISCUSSION THREAD

**Discussion by html5-ninja** [Page-Replica/page-replica#3] [Show and tell]
If you have any questions, need assistance, or have any feedback regarding Page-Replica.com, you've come to the right place. Please feel free to start a discussion here.

**Comment by nitelite** [comment]
Why does logging into page-replica.com require read and write access to my Github profile? I would love to try out the service, but I can't give you write access to both my profile and my companies entire Github organization. You should look into what scopes you are requesting and limit them to what is strictly neccesary.

![image](https://github.com/user-attachments/assets/e52c9119-9166-444d-aa91-cb4bd177653f)

**Reply by html5-ninja** [reply]
Nice catch! You are totally right; I will fix it.

**Reply by html5-ninja** [reply]
@nitelite , fixed , thank you for the feedback

**Reply by nitelite** [reply]
Awesome, signed up as soon as I noticed! Looking forward to see if we can optimize our ecommerce store using this for SEO.

**Comment by ynwin1** [comment]
Hi there! Thanks for creating this tool.
I just wanted to ask if it's possible to make the scraping time longer perhaps on my end or on your end. 
Sometimes, the pages take a while to load, and the tool captures the page right before things are available so they usually come back empty. Thanks a ton!

**Reply by html5-ninja** [reply]
hi @ynwin1 thank you for your feedback, technically the tame frame for rendering should be enough even for slow web app , but let me check first i will get back to asap

**Reply by ynwin1** [reply]
Please take your time. If it's not possible at all, that's completely OK. I'm just a newbie and it's very possible that things aren't working well on my end causing the pages to load longer. Thanks for taking a look into it!

**Reply by html5-ninja** [reply]
I think solved the problem , I will push a fix during the day to update page-replica.com :) ,  keep you updated

**Reply by html5-ninja** [reply]
After a closer investigation, it appears you’re using @emotion/css.
To ensure CSS renders properly in the DOM, please disable Emotion's "speedy" mode as recommended here: https://emotion.sh/docs/ssr#puppeteer.

Could you try this adjustment and re-render your pages? It could also be a great case to document.

**Reply by ynwin1** [reply]
Hi there! Thanks for the reply.
I have tried the above, and unfortunately, it still doesn't work for other pages. I pre-rendered another page, and it returned a page with empty content.  I think the root cause of this is centred around the page being captured before the content arrives, so at the time of capture, there is nothing yet and returns an empty page. You can check this out [here](https://cache.page-replica.com/DCeBnQozGisvr2ixC/www.romanizedmm.com/song/AthelKwelPyaTike).

I'll try to do some optimization on my end first. Then, I'll give it another try. Thanks for all the help!

**Reply by html5-ninja** [reply]
sure I will investigate on my side too

**Comment by diegohb** [comment]
Unsure if this is the right place but I create a feature request for hosted plan at: https://github.com/Page-Replica/page-replica/issues/7

**Reply by admin** [reply]
Awesome, thank you for your great work
I’ll let you know when it’s implemented.
